cpp/src/aztec/honk/composer/standard_honk_composer.test.cpp

```diff
@@ -19,90 +20,108 @@ namespace test_standard_honk_composer {
  */
 TEST(standard_honk_composer, test_sigma_and_id_correctness)
 {
+    auto test_permutation = [](StandardHonkComposer& composer) {
```

Comment:
This was put into a lambda so it would be possible to test different circuit configurations, but so far we still use the same configuration as before (with a public variable)